packages/round-manager/src/features/api/services/grantApplication.ts

```diff
@@ -1,19 +1,29 @@
-import { ethers } from "ethers"
-import { api } from ".."
-import { roundImplementationContract } from "../contracts"
-import { GrantApplication, MetadataPointer } from "../types"
-import { checkGrantApplicationStatus, fetchFromIPFS, graphql_fetch, pinToIPFS } from "../utils"
-
-
-const updateApplicationList = (applications: GrantApplication[], roundId: string, provider: any): Promise<string> => {
+import { ethers } from "ethers";
+import { api } from "..";
+import { roundImplementationContract } from "../contracts";
+import { GrantApplication, MetadataPointer } from "../types";
+import {
+  checkGrantApplicationStatus,
+  fetchFromIPFS,
+  graphql_fetch,
+  pinToIPFS,
+} from "../utils";
+
+const updateApplicationList = (
+  applications: GrantApplication[],
+  roundId: string,
+  provider: any
+): Promise<string> => {
+  /* ESLint ignore is fine since we handle the throws*/
```

Comment:
Agree, could you please add that to the board and assign me? I've resolved the other types of issues, and some of the `any`s and non-null assertions as well, and will pair with someone with a bit more knowledge of the history of the codebase on the rest, since it's mostly about specifying types.

Reply:
created a quick issue here https://github.com/gitcoinco/grants-round/issues/374